apps/docs/functions/src/ai/rate-limit.ts

```
@@ -0,0 +1,77 @@
+  summary: { maxRequests: 20, windowMs: 60 * 60 * 1000 }, // 20/hour
+  research: { maxRequests: 20, windowMs: 60 * 60 * 1000 }, // 20/hour
+  ask: { maxRequests: 50, windowMs: 60 * 60 * 1000 }, // 50/hour for RAG
+};
+
+/**
+ * Check if user has exceeded rate limit for a feature
+ * @returns true if request can proceed, false if rate limited
+ */
+export async function checkRateLimit(
+  userId: string,
+  feature: string
```

ESLint

[error] Insert `,`

(prettier/prettier)